meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -519,6 +561,13 @@ python __anonymous() {
             prev_size=prev_part_info.get("size_KiB"),
             prev_fs_part_number=prev_fs_part_number,
         )
+        if _process_bool_var(d, part_name, "FILL_STORAGE"):
+            part_info["size_KiB"], part_info["size_MiB"] = _extend_part_to_end_of_storage(
+                d,
+                part_name,
+                size_KiB=part_info["size_KiB"],
```

Comment:
Is there a reason the `size_KiB` and `offset_KiB` are keyword args while the other two are positional args? It doesn't look like they have default values and it's no less clear what these parameters are than `d` or `part_name` given the way this function is invoked.

Reply:
I suppose it was just because there are a bunch of functions in this chunk of code whose first two parameters are `d` and `part_name` (in that order) so I thought those two args would need less explanation that the other two.

I can see that the inconsistency is confusing though. I'll switch to just using positional args in this invocation.

Reply:
Switched the keyword args to positional args in https://github.com/ARMmbed/meta-mbl/pull/674/commits/767abb3d03d247c01085888157438083a029b385.